envs/moc/zero/operate-first/jupyterhub-stage.yaml

```
@@ -0,0 +1,20 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: opf-jupyterhub-stage
+spec:
+  destination:
+    name: zero
+    namespace: opf-jupyterhub-stage
+  project: operate-first
+  source:
+    path: odh/overlays/moc/zero/jupyterhub-stage
+    repoURL: https://github.com/operate-first/apps.git
+    targetRevision: HEAD
+  syncPolicy:
+    automated:
+      prune: true
+      selfHeal: true
```

```suggestion
    automated: {}
```

As this is for stagging namespace, can we keep this just automated for deployment without selfheal or prune?